apps/site/assets/css/_alerts.scss

```diff
@@ -126,7 +126,7 @@
   position: absolute;
   top: $base-spacing;
 
-  [class*="c-svg__icon"] {
+  [class*='c-svg__icon'] {
```

Comment:
If the current codebase prefers double quotes, could we configure Stylelint to prefer them and avoid the diff noise of converting the quote style? Or do we have a mix of doubles and singles currently?

Reply:
Somehow, we had a mix of both!

Reply:
Fantastic 😆 Ignore me, then